src/components/AssetAccounts/AssetAccountRow.tsx

```diff
@@ -43,20 +43,33 @@ const stackSpacingMd4 = { base: 2, md: 4 }
 const flexDisplayLgFlex = { base: 'none', lg: 'flex' }
 const flexDisplayMdFlex = { base: 'none', md: 'flex' }
 
-export const AssetAccountRow = ({
-  accountId,
-  assetId,
-  showAllocation,
-  isCompact,
-  ...rest
-}: AssetAccountRowProps) => {
+export const AssetAccountRow = (props: AssetAccountRowProps) => {
+  const { accountId, assetId, showAllocation, isCompact, simpleGridProps } = useMemo(() => {
+    const { accountId, assetId, showAllocation, isCompact, ...simpleGridProps } = props
+    return {
+      accountId,
+      assetId,
+      showAllocation,
+      isCompact,
+      simpleGridProps,
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, Object.values(props))
```

Comment:
@woodenfurniture noticed this guy has 5 references in develop already but do we need this? i.e https://github.com/shapeshift/web/pull/6316#discussion_r1505280335

Reply:
Looking into this again, I believe I get the intent - every render will evaluate to `[...propsValuesAsArray]` expression which will be used instead of exlicit `[prop1, prop2]` deps array 🧠

Reply:
That's tidy. We might want to add a comment with the `eslint-disable` to explain why, so it doesn't come up again.